corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -750,6 +750,14 @@ class SimplifiedSyncLog(AbstractSyncLog):
     closed_cases = SetProperty(unicode)
     extensions_checked = BooleanProperty(default=False)
 
+    _purged_cases = set()
```

Comment:
Is this meant to be a class-level mutable set? Would it work to set this to `None` instead so the class attribute is immutable? Seems like bad things would happen if this set instance somehow got an item added to it, and as far as I can tell this instance is never used.